docs/en/setup/backend/backend-token-auth.md

```diff
@@ -0,0 +1,44 @@
+# Token Authentication
```

Comment:
I think this doc should be linked from main doc. At least, from following places
1. https://github.com/apache/skywalking/blob/master/docs/en/setup/backend/backend-setup.md#advanced-feature-document-link-list
1. Quick link from https://github.com/apache/skywalking/blob/master/docs/README.md